napari/components/_layer_slicer.py

```diff
@@ -74,7 +75,7 @@ def __init__(self):
         """
         self.events = EmitterGroup(source=self, ready=Event)
         self._executor: Executor = ThreadPoolExecutor(max_workers=1)
-        self._force_sync = True
+        self._force_sync = get_settings().experimental.async_slicing
```

Comment:
I think we want to force sync all the time when the setting is off, right?

```suggestion
        self._force_sync = not get_settings().experimental.async_slicing
```

Reply:
Oh! Of course! I was so focused on whether things were passing correctly, I lost track of accuracy 😬